packages/cli/README.md

```diff
@@ -69,6 +69,53 @@ OPTIONS
 ```
 <!-- commandsstop -->
 
+### Options
+#### `-d, --dynamic`
+Dynamically generate examples.
+
+Prism uses [JSON-schema-faker](https://github.com/json-schema-faker/json-schema-faker) under the hood to generate dynamic examples.
+
+Because the OpenAPI spec [allows extensions](https://swagger.io/docs/specification/openapi-extensions/) using a `x-` prefix on the custom keyword and prism extends [JSON-schema-faker](https://github.com/json-schema-faker/json-schema-faker) with [faker.js](https://github.com/marak/Faker.js/), you can add the `x-faker` keyword with a faker API method to provide more specific mocking.
+
+**Example:**
+
+Schema:
+```
+{
+  "type": "object",
+  "properties": {
+    "ip": {
+      "type": "string",
+      "example": "192.168.1.1",
+      "x-faker": "internet.ip"
+    },
+  },
+}
+```
+
+Result:
+```
+{
+  "ip": "255.255.0.0"
+}
+```
+
+#### `-h, --host=host`
+Host that prism will listen to.
+
+Default: `127.0.0.1`
+
+#### `-m, --multiprocess`
+Fork the http server from the CLI
+
+#### `-p, --port=port`
+**Required**
+
+Port that prism will run on.
+
+Default: `4010`
+
+
```

Comment:
Unfortunately this is not the right section — this gets generated automatically from oclif when we're publishing so this would just get lost. We need to write it in another section; this should probably go to the main README? @philsturgeon

Reply:
Time to start a docs/ folder using Stoplight Studio to write Markdown docs, just like Spectral has. 

We need an "Examples and Response Generation" section explaining: 

1. Static Examples
2. Example Selector
3. Dynamic Examples
4. Dynamic Examples (with Faker)

Bigger job though so you know the drill, make an issue and shove it in next sprint :)

Reply:
I'm resolving this since we have a card for documentation.